chapters/renderpass.txt

```diff
@@ -3407,7 +3406,59 @@ pass.
   * [[VUID-VkRenderPassBeginInfo-renderPass-00904]]
     pname:renderPass must: be <<renderpass-compatibility,compatible>> with
     the pname:renderPass member of the sname:VkFramebufferCreateInfo
-    structure specified when creating pname:framebuffer.
+    structure specified when creating pname:framebuffer
+ifndef::VK_VERSION_1_1,VK_KHR_device_group[]
+  * pname:renderArea.offset.x must: be greater or equal to 0
```

Comment:
"greater *than* or equal to" (and also below).

Reply:
done